index.html

```diff
@@ -9,11 +9,12 @@
     <meta content='https://raw.githubusercontent.com/sbstjn/timesheet.js/master/screen.png' property='og:image'>
     <meta content='669118929' property='fb:admins'>
     <title>Timesheet.js - Open time tables with HTML, JavaScript and CSS …</title>
-    <script src="javascripts/timesheet-ec6cdbbf.js" type="text/javascript"></script><script src="javascripts/main-d80740b0.js" type="text/javascript"></script>
-    <link href="http://fonts.googleapis.com/css?family=Oxygen+Mono:400,600,800" rel="stylesheet" type="text/css" />
-    <link href="http://fonts.googleapis.com/css?family=Open+Sans:400,600,800" rel="stylesheet" type="text/css" />
-    <link href="http://fonts.googleapis.com/css?family=Signika+Negative:300,400,600,700" rel="stylesheet" type="text/css" />
-    <link href="stylesheets/all-4d3cb721.css" rel="stylesheet" type="text/css" />
+    <script src="javascripts/timesheet-ec6cdbbf.js"></script>
+    <script src="javascripts/main-d80740b0.js"></script>
+    <link href="https://fonts.googleapis.com/css?family=Oxygen+Mono:400,600,800" rel="stylesheet" />
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,600,800" rel="stylesheet" />
+    <link href="https://fonts.googleapis.com/css?family=Signika+Negative:300,400,600,700" rel="stylesheet" />
+    <link href="stylesheets/all-4d3cb721.css" rel="stylesheet" />
```

Comment:
Note that I also removed `type=text/javascript` and `type=text/css` as [they are redundant](https://mathiasbynens.be/notes/html5-levels#level-1).